### DISCUSSION TITLE
how to get arctic code vault contributor on my profile

### DISCUSSION THREAD

**Discussion by bossoff** [community/community#19360] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi! Can anyone help on how to get an arctic code vault contributor? I really appreciate any help you can provide.

**Comment by melvincwng** [comment, marked as answer]
@bossoff, unfortunately that badge was only available to get in 2020.

It cannot be obtained now as of today.

**Reply by queenofcorgis** [reply]
+1 to @melvincwng -[ linking the FAQ here](https://archiveprogram.github.com/faq/) if that helps!
